README.md

```diff
@@ -401,8 +408,7 @@ Total: 100 / 100
 ```
 
 Extending the previous example, we can manually set the time of submission through `proxy-time`.
-We can use a [Unix Epoch time converter](https://www.epochconverter.com/) to change the target time
-of March, 4th 2025 at 12:00 PM Pacific Time to milliseconds from the Unix Epoch, `1741118400000`:
+We will use the Unix Epoch time `1741118400000` that we converted above.
```

Comment:
Add the string time in parens after the epoch time.

(This comment is a sibling to an earlier one about the epoch times.)